lua/nvim-tree/explorer/change-dir.lua

```diff
@@ -66,6 +66,7 @@ end
 
 ---@return boolean
 local function should_change_dir()
+  local explorer = core.get_explorer()
```

Comment:
This variable `explorer` is unused and has been identified by the linter: https://github.com/nvim-tree/nvim-tree.lua/actions/runs/18935214550/job/54313031114?pr=3209

This line should be deleted.